src/procedures/proc_list_indexes.c

```diff
@@ -248,6 +256,45 @@ static bool _EmitIndex
 		}
 	}
 
+	//--------------------------------------------------------------------------
+	// index fields options
+	//--------------------------------------------------------------------------
+
+	if(ctx->yield_options) {
+		// {field_name, {field_options}
+		// e.g.
+		// {
+		//  'embedding': {'dimension': 100, 'M': 16, 'efConstruction': 200, 'efRuntime': 100}
```

Comment:
why did you choose to call this section: 'embedding'?

Reply:
embedding is the field name

Reply:
I under this is the field name but the value describes a vector index config, why not call this vector_config or vector_parameters?